### PR TITLE
Changes to de_de

### PR DESCRIPTION
**Pull request by Butterkeks1000**
Translates the latest changes to the de_de.json

### CODE REVIEW

**Comment by Butterkeks1000** [comment]
@DasHomi pls review 🙏

File: src/main/resources/assets/preventer/lang/de_de.json [hunk 1]
```diff
@@ -123,7 +123,7 @@
   "text.autoconfig.PreventerConfig.option.moduleUseInfoGroup.preventStemBreaking_msg": "Verhindere das Abbauen von Stämmen",
   "config.preventer.preventStemBreaking.text": "§4Es wurde verhindert das du diesen Stamm abbaust§4",
 
-  "text.autoconfig.PreventerConfig.option.preventItemFrameBreaking": "Verhindert das Zerstören von Rahmen",
```
Comment (DasHomi) [inline src/main/resources/assets/preventer/lang/de_de.json]: , is missing

**Review by DasHomi — CHANGES_REQUESTED** [review]
,

File: src/main/resources/assets/preventer/lang/de_de.json (hunk 1 above)
Comment (Butterkeks1000) [reply src/main/resources/assets/preventer/lang/de_de.json]: Sehr interessant

**Pull request merged**